examples/hugectr/criteo-hugectr.ipynb

```diff
@@ -174,24 +294,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [],
    "source": [
     "proc = nvt.Workflow(\n",
     "    cat_names=CATEGORICAL_COLUMNS,\n",
     "    cont_names=CONTINUOUS_COLUMNS,\n",
-    "    label_name=LABEL_COLUMNS)\n",
+    "    label_name=LABEL_COLUMNS,\n",
+    "    client = client)\n",
     "\n",
     "# log -> normalize continuous features. Note that doing this in the opposite\n",
     "# order wouldn't make sense! Note also that we're zero filling continuous\n",
     "# values before the log: this is a good time to remember that LogOp\n",
     "# performs log(1+x), not log(x)\n",
-    "proc.add_cont_feature([FillMissing(), Clip(min_value=0), LogOp()])\n",
-    "proc.add_cont_preprocess(Normalize())\n",
+    "proc.add_cont_feature([ops.FillMissing(), ops.Clip(min_value=0), ops.LogOp()])\n",
     "\n",
-    "# categorification with frequency thresholding\n",
-    "proc.add_cat_preprocess(Categorify(freq_threshold=15, out_path=OUTPUT_DATA_DIR))"
+    "# categorification\n",
+    "proc.add_cat_preprocess(ops.Categorify(dtype = np.int64, out_path=stats_path))\n",
+    "# MOD 10M\n",
+    "proc.add_cat_preprocess(ops.HashBucket(10000000))"
```

Comment:
Do we want to use hashing on the categorical features here? Whats the benefit of doing both hashing and categorify as part of the same workflow?

Reply:
After the conversations with Chirayu about DLRM. I wanted to apply MOD 10M, so I am using HashBucket for this purpose as you told me.

I am only applying categorify after it, because if not I cannot get the embedding sizes (`ops.get_embedding_sizes(proc)` was giving me an error with no categorify).

Is there any other way to do it better?

Reply:
Lets wait for Ronay's changes on this (in a future PR) -

Reply:
> Lets wait for Ronay's changes on this (in a future PR) -
@benfred  could you pls review https://github.com/NVIDIA/NVTabular/pull/380?  If @albert17 does not have `list columns` my PR should be good to go for him. I'd be glad if you could review the PR. It does not do list hashing in Categorify bcs of https://github.com/NVIDIA/NVTabular/issues/381.